documentation/slate/source/includes/objects.md

```diff
@@ -264,6 +264,10 @@ A container containing either a [admin](#admin), [poi](#poi), [address](#address
 
 ### Trip
 
+A trip corresponds to a scheduled train circulation (and all its linked real-time informations).
+
+An example : the train doing Paris-Marseille every day at 19h30 is the Trip named 456789.
+
```

Comment:
Ok I see, I made it more generalist https://github.com/CanalTP/navitia/pull/1650/commits/873386eec7fd546b4118c8ac798f9ab856bb56de